pennylane/interfaces/batch/__init__.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This subpackage defines functions for interfacing devices with batch execution

```

Comment:
I was just wondering if we shouldn't refer to it too much as "batch execution", since the batch part is not so super important here, it is more the execution, but it happens to take multiple tapes. 

In future, developers could be very confused why we mention "batch" everywhere...

Reply:
Yes, good point. This is only for now to distinguish it from current interfaces, but ideally it would fully replace them down the line (and lose the 'batch' prefix).